MatrixSDK/Room/Polls/PollAggregator.swift

```diff
@@ -137,8 +135,23 @@ public class PollAggregator {
                                  events: events,
                                  currentUserIdentifier: session.myUserId,
                                  hasBeenEdited: hasBeenEdited)
+    }
+    
+    
+    /// Creates a temporary poll without having the start event yet
+    private func buildPollWithoutStartEvent() {
```

Comment:
It would be nice if we would be able to switch the `Poll` to an optional one as described [here](https://github.com/vector-im/element-ios/pull/7518#issuecomment-1523124000).
What do you think?

Reply:
Done. (see the related element-ios PR).